base/config.c

```diff
@@ -1755,7 +1755,7 @@ int pre_flight_object_check(int *w, int *e) {
 			temp_host->notification_period_ptr = temp_timeperiod;
 			}
 
-		/* check all parent parent host */
+		/* check all parents of host */
```

Comment:
I spent some time thinking about this and this is my best guess, but I could very easily be wrong